lib/SPIRV/SPIRVBuiltinHelper.cpp

```diff
@@ -103,7 +104,8 @@ Value *BuiltinCallMutator::doConversion() {
       Builder.Insert(addCallInst(CI->getModule(), FuncName, ReturnTy, Args,
                                  &Attrs, nullptr, Mangler.get()));
   NewCall->copyMetadata(*CI);
-  NewCall->setAttributes(CI->getAttributes());
+  NewCall->setAttributes(CallAttrs);
```

Comment:
`setAttributes` + `getAttributes` also copies calling convention. Are we sure, that CallAttrs also contains it?

Reply:
Calling convention isn't part of attributes. Calling convention for the new function is already set right after function creation at https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/1088aca0fd5ed0b855881bb52a199497dc86e952/lib/SPIRV/SPIRVUtil.cpp#L287

Reply:
The change at line 107 is the key change of this PR.